src/mono/wasm/wasm.proj

```diff
@@ -104,4 +102,109 @@
           SkipUnchangedFiles="true" />
   </Target>
 
+  <UsingTask TaskName="GenerateWasmVersionFile"
+    AssemblyFile="$(WasmBuildTasksAssemblyPath)"/>
+  <Target Name="BuildWasmRuntimes"
+          Condition="'$(OS)' == 'Windows_NT'"
```

Comment:
this should be used everywhere, not just Windows :)

Reply:
If we really want to build this on windows for some reason, we should move to a cmake based build instead of this.

Reply:
Let's get this in as a first step and then we can switch later.